src/cennznut/v0/contract.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright (C) 2019-2020 Centrality Investments Limited
+//!
+//! # CENNZnut - Contracts
+//!
+//! Delegated smart contract permissioning of CENNZnut for use in CENNZnet
+//!
+
+#![warn(clippy::pedantic)]
+
+#[cfg(not(feature = "std"))]
```

Comment:
This should be done in one top level place e.g. `lib.rs` and `alloc` just imported here

Reply:
Done